src/DataCollectors/TraceDataCollector/VanguardCollector/DynamicCoverageDataCollectorImpl.cs

```diff
@@ -112,8 +112,7 @@ public string GetSessionName()
 
             try
             {
-                var processor = new CodeCoverageRunSettingsProcessor(defaultConfigurationElement);
-                configurationElement = (XmlElement)processor.Process(configurationElement);
+                configurationElement = this.RunSettingsProcessor(configurationElement, defaultConfigurationElement);
```

Comment:
lets have method AddDefaultExclusions

with content:
           var processor = new CodeCoverageRunSettingsProcessor(defaultConfigurationElement);
                configurationElement = (XmlElement)processor.Process(configurationElement);

Could you please add comment to this function with explanation why we are adding it.

I want to make sure that nobody will remove it :)